packages/notebook-extension/src/index.ts

```diff
@@ -708,8 +708,6 @@ function addCommands(app: JupyterLab, services: ServiceManager, tracker: Noteboo
     if (!isEnabled()) { return false; }
     const { notebook } = tracker.currentWidget;
     const index = notebook.activeCellIndex;
-    // Can't run above if we are at the top of a notebook.
-    if (index === notebook.widgets.length - 1) { return false; }
```

Comment:
This got mistakenly refactored  at some point. The intention was to have *different* logic for `Run All Above` and `Run All Below`, so that the command was only enabled if the selected cell *had* cells above or below.